libcloudforensics/gcp.py

```diff
@@ -950,7 +1054,7 @@ def GetOperation(self):
     """Get API operation object for the disk.
 
     Returns:
-       str: An API operation object for a Google Compute Engine disk.
+      dict: An API operation object for a Google Compute Engine disk.
```

Comment:
Were these ACTUALLY strings before? Or was it an error in the docstring?

Reply:
it was an error in doc strings

Reply:
Ok, cool :)